packages/drivers/odsp-driver/src/test/socketTests/socketTests.spec.ts

```diff
@@ -343,5 +381,109 @@ describe("OdspDocumentDeltaConnection tests", () => {
 		);
 		assert(errorReceived.errorType, DriverErrorTypes.genericNetworkError);
 		assert(socket !== undefined && !socket.connected, "socket should be closed");
+		checkListenerCount(socket);
+	});
+
+	it("Multiple connection objects should handle server_disconnect event without clientId", async () => {
+		socket = new ClientSocketMock();
+		const connection1 = await mockSocket(socket as unknown as Socket, async () => {
```

Comment:
#nit you can drop the braces and the return statement and just do `async () =>  OdspDocumentDeltaConnection.create(`